jsonb-generator/src/test/java/io/avaje/jsonb/generator/models/valid/OptionalAccess.java

```diff
@@ -0,0 +1,19 @@
+package io.avaje.jsonb.generator.models.valid;
```

Comment:
This isn't a sufficient test without actually any assert per se.  We pretty much need any feature tested in blackbox test.

I'm sure this breaks the ViewBuilder support because the result of the methodHandle will be an optional - it isn't aware of Optional and does not unwrap it (to null if empty). We need to extend the test to include other properties and include a test that uses the view builder.

Reply:
yeah didn't consider views, without doubt this will definitely break them